javascript/ql/lib/semmle/javascript/security/dataflow/PrototypePollutingAssignmentQuery.qll

```diff
@@ -53,6 +53,8 @@ class Configuration extends TaintTracking::Configuration {
         not replace.getRawReplacement().getStringValue() = ""
       )
     )
+    or
+    node = NodeJSLib::Path::moduleMember(["join", "normalize", "relative", "resolve"]).getACall()
```

Comment:
Add test?

Reply:
Actually this turned out to be wrong and the ExtendCall restriction was the real fix.

Reverted the above line, and added some tests (and another fix) to the ExtendCall restriction.